src/ExtractImportGraph.elm

```diff
@@ -10,62 +10,74 @@ import Dict exposing (Dict)
 import Elm.Syntax.Import exposing (Import)
 import Elm.Syntax.ModuleName exposing (ModuleName)
 import Elm.Syntax.Node as Node exposing (Node)
+import Json.Encode as Encode
 import Review.Project.Dependency as Dependency exposing (Dependency)
 import Review.Rule as Rule exposing (Rule)
 import Set exposing (Set)
 
 
-type alias ProjectContext =
-    { imports : List ( { name : ModuleName, isSourceModule : Bool }, List { name : ModuleName, isDependency : Bool, isSourceModule : Bool } )
-    , dependencyModules : Set ModuleName
-    , nonSourceModules : Set ModuleName
-    }
-
-
-type alias ModuleContext =
-    { imports : List { name : ModuleName, isDependency : Bool, isSourceModule : Bool }
-    , moduleName : ModuleName
-    , dependencyModules : Set ModuleName
-    , nonSourceModules : Set ModuleName
-    }
-
-
-initContext : ProjectContext
-initContext =
-    { imports = [], dependencyModules = Set.empty, nonSourceModules = Set.empty }
-
-
-{-| Reports... REPLACEME
+{-| Extract the import graph of your project's modules.
 
     config =
         [ ExtractImportGraph.rule
         ]
 
+This will generate a JSON with two fields:
+
+  - `graph` - The import graph for your project's modules
+  - `onlineGraph` - A link to [GraphvizOnline](https://dreampuf.github.io/GraphvizOnline/) where you can directly view your graph (might not work if your project is too big though)
```

Comment:
Maybe add like "you can always use the [Graphviz](https://graphviz.org/download/) CLI `dot` if the project is too large" or something in case people aren't familiar with what to do if the online solution doesn't work?

Reply:
Oh this is cool! I didn't even know this! Yeah adding ` | dot -Tsvg -o import-graph.svg` makes it possible to directly create an SVG of it for instance (mostly writing it down for reference for myself).